README.md

```diff
@@ -1,41 +1,63 @@
-# Getting Started with Create React App
+# React App: Bookstore CMS
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+    The Bookstore is a website built using React and Redux. This MVP version of the App allows you to:
+
+        *Display a list of books.
+        *Add a book.
+        *Remove a selected book.
+
+    It has two subpages: "Books" and "Categories."
+
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- React 
+
+## Authors
+👤 **Alaa Shaheen**
+
+- GitHub: [@AlaaShaheen2](https://github.com/AlaaShaheen2)
+- Twitter: [@Alaa Shaheen](https://twitter.com/AlaaShaheen93)
+- LinkedIn: [@Alaa Shaheen](https://www.linkedin.com/in/alaa-shaheen-879140240/)
 
 ## Available Scripts
 
 In the project directory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+Runs the app in the development mode.
 
```

Comment:
- [ ] Please update your Readme, currently, some headings are missing such as  `Getting Started` It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.